core/code/portal_detail_display_tools.js

```diff
@@ -2,16 +2,33 @@
 // hand any of these functions the details-hash of a portal, and they
 // will return pretty, displayable HTML or parts thereof.
 
+window.getPortalHistoryDetails = function (d) {
+  if (!d.history) {
+    return ('<div style="text-align: center; color: #ffce00">History missing</div>')
+  }
+
+  var visited = d.history.visited;
+  var captured = d.history.captured;
+  var scoutControlled = d.history.scoutControlled;
+  var colors = {true:'#03fe03',false:'#ff4a4a'};
+
+  return ('<div style="text-align: center; color: #ffce00">History:'
+  + ' <span id="history_visited" style="color:'+ colors[visited] + '">visited</span> |'
+  + ' <span id="history_captured" style="color:'+ colors[captured] + '">captured</span> |'
+  + ' <span id="history_scout_controlled" style="color:' + colors[scoutControlled]  + '">scout controlled</span>'
+  + '</div>');
```

Comment:
todo: move style to CSS

Reply:
Are you going to enhance this PR?
Or this note is just a task for future?

Reply:
I can do a quick sub PR for this

Reply:
https://github.com/MysticJay/ingress-intel-total-conversion/pull/7